halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/EchoProfileFactory.java

```diff
@@ -68,6 +66,15 @@ protected void setProfile(Profile profile, DeploymentConfiguration deploymentCon
       profile.appendContents(yamlToString(deploymentConfiguration.getName(), profile, new ArtifactWrapper(artifacts)));
     }
 
+    Cis cis = deploymentConfiguration.getCi();
```

Comment:
The plurality of the function call doesn't match the return type, is that intentional?

Reply:
Yeah, unfortunately it's correct like this (though confusing).  It's an artifact of the fact that the deployment configuration field called `ci` is actually of type `Cis` (https://github.com/spinnaker/halyard/blob/e07f6c92b479000b3b7854fda9b5bd8a5eebdafc/halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java#L88)

Reply:
Got it, bummer.